<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_control</class>
 <widget class="QDialog" name="dlg_control">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>94</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Control Panel</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="lyt_main">
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <item>
    <widget class="QPushButton" name="btn_play">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>3ds</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Play the animation.</string>
     </property>
     <property name="text">
      <string>PLAY</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_pause">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>3ds</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Pause playing,
rewinding or recording.</string>
     </property>
     <property name="text">
      <string>PAUSE</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_rewind">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>3ds</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Play the animation backwards.</string>
     </property>
     <property name="text">
      <string>REWIND</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_record">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>3ds</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Record this animation
into sequence of images.</string>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="text">
      <string>RECORD</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_export">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>3ds</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Export recorded sequence
 of images into a video.</string>
     </property>
     <property name="text">
      <string>EXPORT</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_seek">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>3ds</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SEEK</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="sld_seek">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>3ds</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Seek through the animation range.</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>5</number>
     </property>
     <property name="tracking">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
